src/coreclr/dlls/mscoree/mscorwks_unixexports.src

```diff
@@ -15,6 +15,7 @@ g_dacTable
 MetaDataGetDispenser
 
 ; mono
+initialize_unity
```

Comment:
Not a blocker, but could be named better based on both convention and purpose. Maybe `mono_unity_initialize_host_apis`?

Reply:
Done.